harmonizer/src/lib.rs

```diff
@@ -0,0 +1,216 @@
+use deno_core::Op;
```

Comment:
to all the crates I write, I also like adding these configs:
```rust
#![forbid(unsafe_code)]
#![deny(missing_debug_implementations, nonstandard_style)]
#![warn(missing_docs, future_incompatible, unreachable_pub, rust_2018_idioms)]
```

These will mean you will need to document some of your methods and have a top level doc comment.

Reply:
Thanks! Fixed in 12102a9cb1c276ae1efdfe1fff996bf7d884fdae and 0ef75de977c67be50b11111e99c77bcd49816008 with what I hope is a ton of valuable documentation!  Though my first foray into rustdoc so probably fraught with misformatting.